setup.py

```diff
@@ -263,7 +263,8 @@ def copy_data_tree(datadir, destdir):
     install_requires=inst_reqs,
     extras_require={
         'ipython': ['ipython>=2.0'],
-        's3': ['boto3']})
+        's3': ['boto3'],
+        'test': ['boto3', 'packaging']})
```

Comment:
`packaging.version.parse` is really handy for testing. I don't want it to be a runtime requirement.